CONTRIBUTING.md

```diff
@@ -1,57 +1,62 @@
 # Development Environment
 
-Getting setup is easy, you'll need to have [node.js](https://nodejs.org/) installed,
+Getting setup is easy, you'll need to have [Node.js](https://nodejs.org/) installed,
 then simply clone the repository and within the working directory run the following:
 
 ```
-# You only need to run this the first time
+# You only need to run this the first time if you don't already have yarn installed
 sudo npm install -g yarn
 yarn install
 
 # Run this to start the development server and build system
-npm run dev
+yarn run dev
```

Comment:
May I ask what the actual difference is? I used `npm run dev` to set up my environment and it seemed to work fine, what changes when you use `yarn run dev` instead?

Reply:
It works pretty much exactly the same in this case. But, if you're going to install `yarn` right before this, you might as well use it, otherwise there's no point in having it installed in the lines above.